From 1f52f7857bdfd54af3409f0dc4a56ed866582327 Mon Sep 17 00:00:00 2001
From: Fabricio Henrique Ramos <fabriciohenrique.ramos@windriver.com>
Date: Fri, 28 Jan 2022 16:31:00 -0300
Subject: [PATCH] Remove User directive from unit file

By removing:
User=docker-registry
the service now runs as user root which allows ansible to use the
registry without hitting permissions issues.

Signed-off-by: Fabricio Henrique Ramos <fabriciohenrique.ramos@windriver.com>
---
 debian/docker-registry.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/docker-registry.service b/debian/docker-registry.service
index a2feaa6..a4652b3 100644
--- a/debian/docker-registry.service
+++ b/debian/docker-registry.service
@@ -3,7 +3,6 @@ Description=the Docker toolset to pack, ship, store, and deliver content
 After=network.target
 
 [Service]
-User=docker-registry
 Type=simple
 Environment=REGISTRY_STORAGE_DELETE_ENABLED=true
 ExecStart=/usr/bin/docker-registry serve /etc/docker-distribution/registry/config.yml
-- 
2.17.1

